Makefile.am

```diff
@@ -109,14 +109,20 @@ generate_ver = ver="`{ $(srcdir)/scripts/version || echo '$(VERSION)' ; } | sed
 src/version.h: .remake-version-h
 	mkdir -p src
 	$(AM_V_GEN) $(generate_ver); echo "$$ver" > $@
-src/main.c: src/version.h
+src/config_opts.inc:
+	mkdir -p src
+	$(AM_V_GEN) if test -x ./config.status; then \
+	  ./config.status --config; \
+	else echo "(unknown)"; \
+	fi | sed -e 's/\\/\\\\/g' -e 's/"/\\"/g' -e 's/^/"/' -e 's/$$/\\n"/' -e 's/^/#define JQ_CONFIG /' > $@
```

Comment:
`--build-configuration` prints an extra new line for me, maybe change `'s/$$/\\n"/` to `'s/$$/"/`?

Reply:
The format is uncommitted.  It can have noise :)